src/test/java/edu/hm/hafner/analysis/parser/GhsMultiParserTest.java

```diff
@@ -108,7 +108,18 @@ void issue59118() {
     @Test
     void issue66130() {
         Report warnings = parse("issue66130.log");
-        assertThat(warnings).hasSize(2);
+        assertThat(warnings).hasSize(1);
```

Comment:
It was expected that size one would be the first line without col. 

But it is able to find the line with col. And ignores the one without col.

Am I missing something?